src/containers/DataPackageView.js

```diff
@@ -18,8 +19,12 @@ export class DataPackageViewContainer extends React.Component {
   }
 
   componentDidMount(){
-    let url = "https://raw.githubusercontent.com/anuveyatsu/test_data/master/datapackage.json";
+    let baseUrl = "https://bits.staging.datapackaged.com/metadata/";
+    let url = baseUrl + this.props.publisherName + "/" + this.props.packageName + "/_v/latest/datapackage.json";
+    // let url = "https://raw.githubusercontent.com/anuveyatsu/test_data/master/datapackage.json";
     this.props.dataPackageActions.getDataPackage(url);
+    this.props.dataPackageActions
+      .getDataPackageMetaData(this.props.publisherName, this.props.packageName);
```

Comment:
Why it can not be found in the bitstore? All of this data should be inlined in the datapackage.json and README can be found in bitstore